plotly/tests/test_optional/test_opt_tracefactory.py

```diff
@@ -105,3 +105,54 @@ def test_simple_streamline(self):
         self.assertListEqual(strln['data'][0]['x'][0:100],
                              expected_strln_0_100['x'])
 
+
```

Comment:
Can you make sure that:

1. The label argument *in* works as expected
2. The orientation *in* works as expected
3. colorscale *in* works as expected

It'd be best if these were tested separately to keep these unit tests from overlapping as much as possible.